infrastructure/public-api.yaml

```diff
@@ -87,7 +87,11 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
       security:
-        - api_key: []
+        - api_key:
+            Fn::If:
+              - IsLocalDevEnvironment
```

Comment:
I don't think this works as you expect. I had to remove this to get api keys working in the other environments.

Reply:
So the issue here is that for some weird reason, cloud formation does deploy the `public-api`, on subsequent of the stack so this only works first time around (which is fine).